queue/src/main/java/io/kestra/queue/GenericEvent.java

```diff
@@ -0,0 +1,5 @@
+package io.kestra.queue;
+
+public interface GenericEvent {
```

Comment:
I would only nam e it `Event` or `QueueEvent` as it's the parent of all queue events and not a generic vs type event